run.sh

```diff
@@ -8,13 +8,13 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 
-if [ "$1" == "-prod" ]; then
+if [ "$1" = "-prod" ]; then
   echo "Building for production (publishing on eclipse.org/che/docs/). To preview"
   echo -e "the docs locally, run the script ($0) without the '-prod' option.\n"
   docker run --rm -v $(pwd)/src/main:/che-docs:Z eclipse/che-docs sh -c "cd /che-docs; jekyll build --baseurl '/che/docs/'"
 else
   echo "Building and serving the docs for local preview. To build for production"
   echo "(publishing on eclipse.org/che/docs/), run the script with the '-prod' option:"
-  echo -e "$0 -prod\n"
+  echo "$0 -prod\n"
```

Comment:
The `-e` parameter means that `echo` interprets escape sequences (the `\n` in the string). Again, it's not POSIX compatible.

However, we should also remove the escape character:
```suggestion
  echo "$0 -prod"
```